<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZoezenTracker</class>
 <widget class="QMainWindow" name="ZoezenTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZoezenTracker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_fps">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>255</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>255</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_showMode">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1225</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect_Tracker"/>
    <addaction name="actionNew_Template"/>
    <addaction name="actionPose_Recorder"/>
    <addaction name="actionPose_Sender"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionSave_Images"/>
    <addaction name="actionDetect_Corners"/>
    <addaction name="actionTracking"/>
    <addaction name="actionNoDeal"/>
    <addaction name="actionMatch"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect_Tracker"/>
   <addaction name="actionNew_Template"/>
   <addaction name="actionPose_Recorder"/>
   <addaction name="actionPose_Sender"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect_Tracker">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/start-icon.png</normaloff>:/Images/start-icon.png</iconset>
   </property>
   <property name="text">
    <string>Connect Tracker</string>
   </property>
  </action>
  <action name="actionNew_Template">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/New Template.png</normaloff>:/Images/New Template.png</iconset>
   </property>
   <property name="text">
    <string>New Template</string>
   </property>
  </action>
  <action name="actionPose_Recorder">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/Pose Recorder.png</normaloff>:/Images/Pose Recorder.png</iconset>
   </property>
   <property name="text">
    <string>Pose Recorder</string>
   </property>
  </action>
  <action name="actionPose_Sender">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/sender.png</normaloff>:/Images/sender.png</iconset>
   </property>
   <property name="text">
    <string>Pose Sender</string>
   </property>
  </action>
  <action name="actionSave_Images">
   <property name="text">
    <string>Save Images</string>
   </property>
  </action>
  <action name="actionDetect_Corners">
   <property name="text">
    <string>detect Corner</string>
   </property>
  </action>
  <action name="actionTracking">
   <property name="text">
    <string>Tracking</string>
   </property>
  </action>
  <action name="actionNoDeal">
   <property name="text">
    <string>NoDeal</string>
   </property>
  </action>
  <action name="actionMatch">
   <property name="text">
    <string>Match</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
